jobs/cloud_controller_ng/spec

```diff
@@ -1201,6 +1202,10 @@ properties:
     description: "Use co-deployed Redis for rate limiting and metrics. If the Puma webserver is enabled, Redis will automatically be used."
     default: false
 
+  cc.experimental.enable_statsd_metrics:
```

Comment:
From my point of view the `experimental` part does not fit to `enable_statsd_metrics` anymore as this has always been the default. I would move it up, i.e. `cc.enable_statsd_metrics`.